src/entries/popup/pages/home/TokenDetails/TokenDetails.tsx

```diff
@@ -404,6 +434,8 @@ export function TokenDetails() {
     isTestnetChainId({ chainId: token?.chainId }) || !!customAsset
   );
 
+  const hidden = isHidden(token.address, token.chainId);
```

Comment:
same here, isHidden could receive token as param and then inside of it you can use token.uniqueId